parent/build.gradle

```diff
@@ -32,6 +51,10 @@ ext.extraPomInfo = {
         'maven-shade-plugin.version'('3.2.4')
         'maven-surefire-plugin.version'('2.22.2') // Override actual Maven surefire and failsafe version (2.12) to get native support for executing tests on the JUnit Platform (JUnit 5)
         'protoc-jar-maven-plugin.version'('3.11.4')
+        // inject the properties from the required boms
+        directory.get().file(WriteMicronautVersionInfoTask.MICRONAUT_VERSIONS_PROPERTIES_FILE_NAME).asFile.text.readLines()*.split('=', 2).each {
```

Comment:
Can't you move `def micronautVersionInfo = tasks.register(...)` up and use `micronautVersionInfo.flatMap { it.outputDirectory }` here?

Reply:
Done 👍